test/cts/from_nnapi/README.md

```diff
@@ -0,0 +1,99 @@
+# Converted CTS from NNAPI CTS
+
+## Test Generator Tool
+This [test_generator](./test_generator) tool is for converting existed native 
+[NNAPI CTS](https://android.googlesource.com/platform/frameworks/ml/+/refs/tags/android-cts-10.0_r5/nn/runtime/test/specs/) of V1_0, V1_1 and V1_2 versions to these tests for WebNN API.
+
+### NNAPI Operations being map to WebNN API Operations Tables
+* Part I 
+
+The `test_generator` tool has supported to convert those tests of following
+NNAPI operations to the tests for such WebNN API operations of first wave.
+
+| NNAPI                         | WebNN API (first wave ops)    |
+|:------------------------------|:------------------------------|
+| RELU1                         | clamp                         |
+| RELU6                         | clamp                         |
+| CONCATENATION                 | concat                        |
```

Comment:
I didn't find test cases for `concat`?

Reply:
@huningxin Now I've add 6 (passed 6) concat test, please take a look, thanks.

Reply:
looks good, thanks!